.travis.yml

```diff
@@ -27,7 +27,7 @@ android:
 
 install: /bin/true
 
-script: ./gradlew build
+script: ./gradlew check build
```

Comment:
U sure? I know it does some checks, but I wasn't sure it ran the whole check task. I'll remove it then.